packages/shadcn/src/suggestionMenu/SuggestionMenuItem.tsx

```diff
@@ -24,7 +24,7 @@ export const SuggestionMenuItem = forwardRef<
 
     const overflow = elementOverflow(
       itemRef.current,
-      document.querySelector(".bn-suggestion-menu")!
+      document.querySelector(".bn-suggestion-menu, #ai-suggestion-menu")! // TODO
```

Comment:
@matthewlipski this still seems nasty, can't we pass an `elementOverflow` into the `SuggestionMenuItem` to for example? Or find another way to do this at a higher level without `querySelector`?

Reply:
tracking here and in linear; https://github.com/TypeCellOS/BlockNote/pull/1104